### PR TITLE
Copy to clipboard initial implementation

### PR DESCRIPTION
**Pull request by renjithgr**
- Initial implementation of copy to clipboard functionality #7.
- Added storybook to easily develop components.

### CODE REVIEW

**Comment by renjithgr** [comment]
Initial implementation of #7.

File: src/components/code-editor/CodeEditor.js [hunk 1]
```diff
@@ -20,17 +20,31 @@ const CodeEditor = (props) => {
     valueChanged(value);
   };
 
+  const copyToClipboard = () => {
+    navigator.clipboard.writeText(value).then(
+      function () {
+        console.log("Written to the clipboard");
+      },
+      function () {
+        console.log("Writing to the clipboard failed");
```
Comment (jamesgeorge007) [inline src/components/code-editor/CodeEditor.js:29]:
```suggestion
        console.error("Writing to the clipboard failed");
```

**Comment by jamesgeorge007** [comment]
Can we have the migration to `Ace-Editor` in another PR?

**Comment by renjithgr** [comment]
I will close this PR for now since I realized copying to clipboard will interfere with the editor changes. I will raise a PR for ace editor first then implment the copy to clipboard on top of it.

**Pull request closed without merging**